taylor-francis/taylor-francis-proposal-2020-07.md

```diff
@@ -0,0 +1,308 @@
+
+## 3. Target Market
+
+At the beginning of the book, we introduce three personas who characterize our
+audience:
+
+*Amira Khan* completed a master's in library science five years ago and has
+since worked for a small aid organization.  She did some statistics during her
+degree, and has learned some R and Python by doing data science courses online,
+but has no formal training in programming.  Amira would like to tidy up the
+scripts, data sets, and reports she has created in order to share them with her
+colleagues.  These lessons will show her how to do this and what "done" looks
+like.
+
+*Jun Hsu* completed a 4-month data science bootcamp last year after doing a PhD
+in Geology and now works for a company that does forensic audits.  He uses a
+variety of machine learning and visualization packages, and would now like to
+turn some of his own work into an open source project.  This book will show him
+how such a project should be organized and how to encourage people to contribute
+to it.
+
+*Sami Virtanen* became a competent programmer during a bachelor's degree in
+applied math and was then hired by the university's research computing center.
+The kinds of applications they are being asked to support have shifted from
+fluid dynamics to data analysis; this guide will teach them how to build and run
+data pipelines so that they can pass those skills on to their users.
+
+## 4. Competing Titles
+
+*The Turing Way* (https://the-turing-way.netlify.app/), produced by the Turing
+Institute in the UK, is the only up-to-date resource we know of with similar
+breadth.  Other books cover only one topic in more depth than our audience needs
+(e.g., Ray & Ray's *Unix and Linux: Visual QuickStart Guide*), are aimed at
+commercial software developers (any number of books on testing, coding style,
+and dev ops), or go into data science rather than telling readers how to build
+software (e.g., Zhang's upcoming *A Tour of Data Science*).
+
+Our book has broader and deeper coverage, and includes exercises with solutions.
+Our focus on programming best practices (e.g. principles like writing modular,
+reusable, testable code).  Others teach basic Python and R syntax and how to
+complete isolated tasks with particular libraries, but today, people can pick up
+that information from a Google search.  Our content is the bigger picture you
+can't glean from online code snippets or Stack Overflow answers.
+
+We go through the process of building a data science workflow/package from
+scratch.  Most other texts simply present information (like a reference book):
+we tackle a real data science problem from beginning to end, weaving in relevant
+information as we go.
+
+## 5. Format and Timeline
+
+Our manuscript is approximately 470 pages long, with 40 diagrams and other
+graphics and 150 code listing.
+
+## 6. Other Information
+
+-   Dr. Damien B. Irving: climate scientist working for CSIRO in Tasmania.
```

Comment:
```suggestion
-   Dr. Damien B. Irving: post-doctoral researcher in climate science at the University of New South Wales.
```

Reply:
(I'm still based in Tasmania as a visiting scientist at CSIRO, but my employer is now UNSW.)

Reply:
:+1: